File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV2003U00GrdINV2003Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV2003U00GrdINV2003Info
    {
        private String _pkinv2003;
        private String _churiDate;
        private String _churiTime;
        private String _churiBuseo;
        private String _chulgoType;
        private String _exportCode;
        private String _churiSeq;
        private String _jaeryoGubun;
        private String _ipchulType;
        private String _inOutGubun;
        private String _remark;
        private String _inputUser;
        private String _rowState;

        public String Pkinv2003
        {
            get { return this._pkinv2003; }
            set { this._pkinv2003 = value; }
        }

        public String ChuriDate
        {
            get { return this._churiDate; }
            set { this._churiDate = value; }
        }

        public String ChuriTime
        {
            get { return this._churiTime; }
            set { this._churiTime = value; }
        }

        public String ChuriBuseo
        {
            get { return this._churiBuseo; }
            set { this._churiBuseo = value; }
        }

        public String ChulgoType
        {
            get { return this._chulgoType; }
            set { this._chulgoType = value; }
        }

        public String ExportCode
        {
            get { return this._exportCode; }
            set { this._exportCode = value; }
        }

        public String ChuriSeq
        {
            get { return this._churiSeq; }
            set { this._churiSeq = value; }
        }

        public String JaeryoGubun
        {
            get { return this._jaeryoGubun; }
            set { this._jaeryoGubun = value; }
        }

        public String IpchulType
        {
            get { return this._ipchulType; }
            set { this._ipchulType = value; }
        }

        public String InOutGubun
        {
            get { return this._inOutGubun; }
            set { this._inOutGubun = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String InputUser
        {
            get { return this._inputUser; }
            set { this._inputUser = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public INV2003U00GrdINV2003Info() { }

        public INV2003U00GrdINV2003Info(String pkinv2003, String churiDate, String churiTime, String churiBuseo, String chulgoType, String exportCode, String churiSeq, String jaeryoGubun, String ipchulType, String inOutGubun, String remark, String inputUser, String rowState)
        {
            this._pkinv2003 = pkinv2003;
            this._churiDate = churiDate;
            this._churiTime = churiTime;
            this._churiBuseo = churiBuseo;
            this._chulgoType = chulgoType;
            this._exportCode = exportCode;
            this._churiSeq = churiSeq;
            this._jaeryoGubun = jaeryoGubun;
            this._ipchulType = ipchulType;
            this._inOutGubun = inOutGubun;
            this._remark = remark;
            this._inputUser = inputUser;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV2003U00GrdINV2003Info.java
package nta.med.data.model.ihis.invs;

import java.util.Date;

public class INV2003U00GrdINV2003Info {
	private Double pkinv2003;
	private Date churiDate;
	private String churiTime;
	private String churiBuseo;
	private String chulgoType;
	private String exportCode;
	private Double churiSeq;
	private String jareyoGubun;
	private String remark;
	public INV2003U00GrdINV2003Info(Double pkinv2003, Date churiDate, String churiTime, String churiBuseo,
			String chulgoType, String exportCode, Double churiSeq, String jareyoGubun, String remark) {
		super();
		this.pkinv2003 = pkinv2003;
		this.churiDate = churiDate;
		this.churiTime = churiTime;
		this.churiBuseo = churiBuseo;
		this.chulgoType = chulgoType;
		this.exportCode = exportCode;
		this.churiSeq = churiSeq;
		this.jareyoGubun = jareyoGubun;
		this.remark = remark;
	}
	public Double getPkinv2003() {
		return pkinv2003;
	}
	public void setPkinv2003(Double pkinv2003) {
		this.pkinv2003 = pkinv2003;
	}
	public Date getChuriDate() {
		return churiDate;
	}
	public void setChuriDate(Date churiDate) {
		this.churiDate = churiDate;
	}
	public String getChuriTime() {
		return churiTime;
	}
	public void setChuriTime(String churiTime) {
		this.churiTime = churiTime;
	}
	public String getChuriBuseo() {
		return churiBuseo;
	}
	public void setChuriBuseo(String churiBuseo) {
		this.churiBuseo = churiBuseo;
	}
	public String getChulgoType() {
		return chulgoType;
	}
	public void setChulgoType(String chulgoType) {
		this.chulgoType = chulgoType;
	}
	public String getExportCode() {
		return exportCode;
	}
	public void setExportCode(String exportCode) {
		this.exportCode = exportCode;
	}
	public Double getChuriSeq() {
		return churiSeq;
	}
	public void setChuriSeq(Double churiSeq) {
		this.churiSeq = churiSeq;
	}
	public String getJareyoGubun() {
		return jareyoGubun;
	}
	public void setJareyoGubun(String jareyoGubun) {
		this.jareyoGubun = jareyoGubun;
	}
	public String getRemark() {
		return remark;
	}
	public void setRemark(String remark) {
		this.remark = remark;
	}
	
}
